lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1132,15 +1130,15 @@ def set_vm_id
       @vm_id ||= fetch_gce_metadata('instance/id') if @platform == Platform::GCE
       @vm_id ||= ec2_metadata['instanceId'] if @platform == Platform::EC2
     rescue StandardError => e
-      @log.error 'Failed to obtain vm_id: ', error: e
+      log.error 'Failed to obtain vm_id: ', error: e
```

Comment:
Curious: How did we decide which cases are `log`, and which cases are `$log`?

Reply:
It's hidden in the extra part of the commit message on https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/pull/268/commits/63387b523f3ee703a2ed0cf702d14b87891500cc. Frankly, I'd prefer it as a code comment in `configure`.